spec/ruby-push-notifications/mpns/mpns_connection_spec.rb

```diff
@@ -0,0 +1,46 @@
+
+module RubyPushNotifications
+  module MPNS
+    describe MPNSConnection do
+
+      describe '::post' do
+
+        let(:body) { 'abc' }
+        let(:headers) {
+          {
+            'x-notificationstatus' => ['Received'],
```

Comment:
Why are you using this arrays here? Header values are just strings, not arrays of strings, right? https://msdn.microsoft.com/en-us/library/windows/apps/ff941100(v=vs.105).aspx#BKMK_CustomHeaders